source/dsl/DangerUtilsDSL.ts

```diff
@@ -5,7 +5,7 @@
 export interface DangerUtilsDSL {
 
   /**
-   * Creates an HTML link.
+   * Creates a HTML link.
```

Comment:
hah - great point, I just asked my wife, she also agrees with you - instead, I'll re-write the sentence :D